static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -135,11 +126,6 @@ const pageShouldHideReaderRevenue = () =>
 const shouldShowEpic = (test: EpicABTest): boolean => {
     const onCompatiblePage = test.pageCheck(config.get('page'));
 
-    const storedGeolocation = geolocationGetSync();
-    const inCompatibleLocation = test.locations.length
```

Comment:
The `locations` field is now on the `EpicVariant` model, and checked from the variants' `canRun` function